lib/presentation/extensions/send_file_extension.dart

```diff
@@ -505,4 +505,8 @@ extension SendFileExtension on Room {
   Future<Size> _calculateImageBytesDimension(Uint8List bytes) {
     return Image.memory(bytes).calculateImageDimension();
   }
+
+  bool isRoomEncrypted() {
```

Comment:
could you move it to room please ? Would be useful for encrypted icon for example

Reply:
it's already in the room extension

Reply:
year and on Room :))